src/Illuminate/Database/Schema/Builder.php

```diff
@@ -46,6 +46,14 @@ class Builder
      */
     public static ?int $defaultTimePrecision = 0;
 
+    /**
+     * The default id key type.
+     *
+     * @var string
```

Comment:
It would be nice if you could narrow this further down
```suggestion
     * @var 'int'|'uuid'|'ulid'
```

Reply:
Maybe, this could be handled via an Enum